ansible-ipi-install/roles/node-prep/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# vars file for node-prep
+package_list: ['firewalld','libvirt', 'qemu-kvm', 'mkisofs', 'python3-devel', 'jq', 'ipmitool']
```

Comment:
maybe we don't need mkisofs anymore? https://github.com/metal3-io/baremetal-operator/pull/310

Reply:
yep will remove